sonar-python-plugin/src/main/java/org/sonar/plugins/python/indexer/SonarQubePythonIndexer.java

```diff
@@ -36,13 +37,12 @@ public SonarQubePythonIndexer(List<InputFile> files) {
 
   @Override
   public void buildOnce(SensorContext context) {
+    PerformanceMeasure.Duration duration = PerformanceMeasure.start("ProjectLevelSymbolTable");
```

Comment:
do we do it only for SQ / SC? I think we could add this statistics also for SonarLint

Reply:
From my understanding the performance monitoring makes mostly sense for monitoring on Peach, thus in SQ context.
Although there could be more usages to this and it might make sense to have it in SL context, I felt it's not necessary for now (optimizing symbol table construction for SQ is essentially the same as doing it for SL as of today).
If you think it's worth having it nevertheless, I can still add it though.

Reply:
ok makes sense. Maybe we can add a small comment to explain why we only measure SQ / SC